discovery/spec.md

```diff
@@ -365,775 +244,693 @@ The following sections define the attributes that appear in a Service entity.
     }
     ````
 
-##### specversions
+#### channel
 
-- Type: Array of `String` values
-- Description: CloudEvents
-  [`specversions`](../cloudevents/spec.md#specversion)
-  that can be used for events published for this service.
+- Type: String
+- Description: A string that can be used to correlate Endpoints. Any Endpoints
+  within an instance of a Discovery Service that share the same `channel` value
+  MUST have some relationship. This specification does not define that
```

Comment:
If we do not define what `channel` means, I would lean towards `channel` being either a documented extension that defines one way to interpret `channel`, or drop it entirely and vendors can add private extensions to the api.

Reply:
Would it help if we made it clear that while we leave it pretty open, we expect it to be used to help identify Endpoints for a shared messaging bus instance?